### DISCUSSION TITLE
Dumb question, but why don't i see any events

### DISCUSSION THREAD

**Discussion by r3wt** [blakeblackshear/frigate#3162] [Ask A Question]
Do i have to turn on recording? For all 3 of my cameras, all i see is the last tracked object. and when clicking on it, it just takes me to an empty event table. This is not explained at all in the documentation. Please clarify. I currently have the recording disabled as i do not want to bash my nvme drive with that activity. I have to finish setting up my NAS and get it mounted before i will setup recording. but for now i am puzzled why the events aren't working.

**Comment by NickM-27** [comment]
For an event to be created either snapshots or recordings needs to be enabled. We really can't offer up more guidance without seeing the config file. For example if you defined a zone or a filter then that would affect that behavior.

**Reply by r3wt** [reply]
Ok, thanks for that info. That's fine, and what I suspected based on the cryptic comments in the config file, but again the documentation doesn't make this clear at all.

**Reply by NickM-27** [reply]
Gotcha, I'll work on updating the docs to show this info
